src/applications/simple-forms/20-10207/pages/medicalTreatment.js

```diff
@@ -0,0 +1,64 @@
+import {
+  addressNoMilitarySchema,
+  addressNoMilitaryUI,
+  currentOrPastDateSchema,
+  currentOrPastDateUI,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { VaTextInputField } from 'platform/forms-system/src/js/web-component-fields';
+import { MedicalTreatmentViewField } from '../components/MedicalTreatmentViewField';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    ...titleUI(
```

Comment:
titleUI pattern also now accepts functions thanks to your previous suggestion👍
e.g.
```
...titleUI(
   ({ formData }) => ..., 
   ({ formData }) => ...
)
```
which seems like a great candidate for these pages

Reply:
Thanks, Robert!  I'm gonna refactor for this after the build-f/e step.